### PR TITLE
added a monitoring plugin to see health of peers and channels

### PR DESCRIPTION
**Pull request by renepickhardt**
added a small monitoring plugin

### CODE REVIEW

File: monitor/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Monitor plugin
+
+Monitors the health of your peers and helps you to decide if you might want to close a channel
+
+## Installation
+
+For general plugin installation instructions see the repos main
```
Comment (cdecker) [inline monitor/README.md]:
```suggestion
For general plugin installation instructions see the repo's main
```

File: monitor/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+[README.md](https://github.com/lightningd/plugins/blob/master/README.md#Installation)
+
+## Example Usage
+
+Unfortunately the python plugin framework doesn't pretty-print, nor does
+lightning-cli, so best viewed with -H (sorry for all the slasshes in the output. nothing I can do about it at this point):
```
Comment (cdecker) [inline monitor/README.md]: Wouldn't piping it into `jq` or `json_pp` be better?

File: monitor/monitor.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+This one is adapted by Rene Pickhardt and aimed to help you identify inactive channels quickly
+
+"""
+
+from lightning import Plugin, Millisatoshi
+import lightning
+import json
+
+plugin = Plugin(autopatch=True)
```
Comment (cdecker) [inline monitor/monitor.py]: `autopatch=True` is now the default, so no need to specify it all the time :-)
It uses the `LIGHTNINGD_PLUGIN` environment variable to decide whether we are running as a plugin or not.

File: README.md [hunk 4]
```diff
@@ -101,3 +102,4 @@ your environment.
 [graphql-spec]: https://graphql.org/
 [autoreload]: https://github.com/lightningd/plugins/tree/master/autoreload
 [lightning-qt]: https://github.com/darosior/pylightning-qt
+[monitor]:https://github.com/renepickhardt/plugins/tree/master/monitor
```
Comment (cdecker) [inline README.md]:
```suggestion
[monitor]: https://github.com/renepickhardt/plugins/tree/master/monitor
```

File: monitor/README.md (hunk 2 above)
Comment (renepickhardt) [reply monitor/README.md]: yes but not everyone has those installed / knows to use them. I wrote the plugin for a person who wanted to have as little explanation as possible I was afraid that installing jq would be too much of a hussle

File: monitor/monitor.py (hunk 3 above)
Comment (renepickhardt) [reply monitor/monitor.py]: good to know. will leave it here to save another git commit

**Comment by renepickhardt** [comment]
accepted your suggestions

**Comment by cdecker** [comment]
Since the issue with .gitmodules has made this pretty much unmergeable, and I butchered the commit history, I re-opened this as PR #46

**Pull request closed without merging**